lib/signal/postSignals.ts

```
@@ -141,3 +142,32 @@ export const updatePostState = (postID: Post["id"], state: Task["state"]) => {
 };
+
+export const sortPosts = (
+  criterion: SortCriterion,
+  direction?: SortDirection
+) => {
+  const sortedPosts = [...postSignal.value].sort((a, b) => {
+    let comparison = 0;
+
+    switch (criterion) {
+      case "creation-time":
+        const dateA = new Date(a.createdAt);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)